agent/src/softwarecontaineragent.cpp

```diff
@@ -48,7 +48,6 @@ bool SoftwareContainerAgent::triggerPreload()
 inline bool SoftwareContainerAgent::isIdValid (ContainerID containerID)
 {
     return ((containerID < UINT32_MAX)
-            && (containerID >= 0)
```

Comment:
I guess if we have code that ever does this it would in turn generate a warning from the compiler that we're passing a signed int to this function, if it's even possible.